bolts/data/datasets/audio/ecoacoustics.py

```diff
@@ -128,14 +171,44 @@ def _download_files(self) -> None:
         self.root.mkdir(parents=True, exist_ok=True)
         self.base_dir.mkdir(parents=True, exist_ok=True)
 
-        urls = [
-            "https://zenodo.org/record/1255218/files/AvianID_AcousticIndices.zip",
-            "https://zenodo.org/record/1255218/files/EC_BIRD.zip",
-            "https://zenodo.org/record/1255218/files/UK_BIRD.zip",
-        ]
+        for finfo in self._INDICES_URL_MD5_LIST:
+            if not self._check_integrity(finfo.filename, finfo.md5):
+                download_and_extract_archive(
+                    url=finfo.url, download_root=str(self.base_dir), md5=finfo.md5
+                )
+
+        for finfo in self._URL_MD5_LIST:
+            if not self._check_integrity(finfo.filename, finfo.md5):
+                self.download_and_extract_archive_jar(
+                    url=finfo.url, download_root=str(self.base_dir), md5=finfo.md5
+                )
+
+    def download_and_extract_archive_jar(
+        self,
+        url: str,
+        download_root: str,
```

Comment:
```suggestion
        download_root: str,
```
just replace this with`self._base_dir`?